changelog.sh

```diff
@@ -6,4 +6,9 @@ base="./changelog.d"
 
 new_item="${base}/$(date '+%Y%m%d_%H%M%S')_$(git rev-parse --abbrev-ref HEAD)_scriv.md"
 cp "${base}/new_fragment.md.j2" ${new_item}
-$EDITOR $new_item
+if [ -z "$EDITOR" ]; then
+  echo "\$EDITOR is not set or empty"
+  echo "edit $new_item manually"
```

Comment:
Ah, no, I'd like this to be the other way around. Ask people to set their editor or refuse to do work. I don't want people to implement different workflows here.

Reply:
It would be nice if the script could check the return status of the editor process and discard the changelog if it returned non-zero. This would be more consistent with other tools that spawn editors like git (e.g. when spawning an editor to write a commit message, git will abort creating the commit if the commit message file is unchanged or if the editor exits non-zero).